Whoaverse/Whoaverse/Models/AddSubmissionSelfpost.cs

```diff
@@ -34,8 +34,8 @@ public class AddSubmissionSelfpost
 
 
         [Required(ErrorMessage = "Message title is required. Please fill this field.")]
-        [StringLength(200, ErrorMessage = "Submission title is limited to 200 characters.")]
-        public string Title { get; set; }
+        [StringLength(200, ErrorMessage = "Submission title must be at least 20 and is limited to 200 characters long.", MinimumLength = 20)]
+        public string Linkdescription { get; set; }
```

Comment:
It looks like there was a getter/setter for "Title", but now there is one for "Linkedescription" instead. Is this intended?